### PR TITLE
[K32W0] Update Docker image

### PR DESCRIPTION
**Pull request by doru91**
#### Problem
What is being fixed? 
NXP refactoring around SDK naming.

#### Change overview
Change SDK env variable name.

#### Testing
How was this tested? (at least one bullet point required)
* not tested yet

### CODE REVIEW

**Comment by doru91** [comment]
Not sure if the version update is done correctly by changing integrations/docker/images/chip-build/version. Please double check.

File: integrations/docker/images/chip-build/version [hunk 1]
```diff
@@ -1 +1,2 @@
 0.5.28 Version bump reason: [Ameba] Decompress toolchain in Docker
```
Comment (Damian-Nordic) [inline integrations/docker/images/chip-build/version:1]: You should remove the old line.

Comment (doru91) [reply integrations/docker/images/chip-build/version]: Thanks, solved this.

**Review by Damian-Nordic — APPROVED** [review]

**Comment by andy31415** [comment]
fast track: platform specific docker change

**Pull request merged**